tests/AppBundle/Controller/AdherentControllerTest.php

```diff
@@ -207,6 +207,50 @@ public function testEditAdherentInterests()
         }
     }
 
+    public function testAdherentChangePassword()
+    {
+        $this->authenticateAsAdherent($this->client, 'carl999@example.fr', 'secret!12345');
+
+        $crawler = $this->client->request(Request::METHOD_GET, '/espace-adherent/mon-profil/changer-mot-de-passe');
+
+        $this->assertCount(1, $crawler->filter('input[name="adherent_change_password[old_password]"]'));
+        $this->assertCount(1, $crawler->filter('input[name="adherent_change_password[password][first]"]'));
+        $this->assertCount(1, $crawler->filter('input[name="adherent_change_password[password][second]"]'));
+
+        // Submit the profile form with invalid data
```

Comment:
I don't understand the need, here the test have a response with errors (due to empty string), how could the password change then (error means no persistence possible). Testing otherwise would mean we would test symfony flow, weird.